server/static.go

```diff
@@ -35,6 +36,6 @@ func serveFile(file string, c echo.Context) error {
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	http.ServeContent(c.Response(), c.Request(), stat.Name(), stat.ModTime(), f)
+	http.ServeContent(c.Response(), c.Request(), stat.Name(), stat.ModTime(), f.(io.ReadSeeker))
```

Comment:
Add a comment here to justify casting
```suggestion
	http.ServeContent(c.Response(), c.Request(), stat.Name(), stat.ModTime(), f.(io.ReadSeeker)) // returned file always comes from embed or os.DirFS
```